src/scipp/experimental/plotting/wrappers.py

```diff
@@ -24,6 +24,24 @@ def _to_data_array(obj):
     return out
 
 
+def _convert_if_not_none(x, unit):
+    if x is not None:
+        return to_unit(x, unit=unit)
+    return x
+
+
+def _preprocess(obj, crop):
+    out = _to_data_array(obj)
+    crop = {} if crop is None else crop
```

Comment:
I saw this somewhere above as well.

Reply:
It's also in `Figure`, but one can make a `Figure` without calling `plot`...